utils/frame/remote-externalities/src/lib.rs

```diff
@@ -375,49 +376,62 @@ impl Builder {
 	}
 }
 
-#[cfg(feature = "remote-test")]
 #[cfg(test)]
-mod tests {
-	use super::*;
+mod test_prelude {
+	pub use super::*;
```

Comment:
Best limit the visibility of these items.

```suggestion
	pub(crate) use super::*;
```
…and the same for the others.